modules/argocd-builds-sample.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-builds-sample_{context}"]
+= Builds
+
+You can use ArgoCD to deploy a global Build configuration. The `./builds/base` directory in the ArgoCD samples repository contains an example global Build configuration that is launched by using a link:https://kustomize.io/[Kustomize] application.
+
+.ArgoCD Build configuration sample (`build.yaml`)
+----
+---
+apiVersion: config.openshift.io/v1
+kind: Build
+metadata:
+  name: cluster
+  annotations:
+    argocd.argoproj.io/sync-options: Prune=false
```

Comment:
This won't be needed as pruning it's not being enabled when creating the Argo CD App.